packages/aws-cdk-lib/aws-certificatemanager/lib/certificate.ts

```diff
@@ -103,6 +103,36 @@ export interface CertificateProps {
   readonly certificateName?: string
 }
 
+/**
+ * Certificate Manager key algorithm
+ *
+ * If you need to use an algorithm that doesn't exist as a static member, you
+ * can instantiate a `KeyAlgorithm` object, e.g: `new KeyAlgorithm('RSA_2048')`.
+ */
+export class KeyAlgorithm {
```

Comment:
Allowed values are: `RSA_1024 | RSA_2048 | RSA_3072 | RSA_4096 | EC_prime256v1 | EC_secp384r1 | EC_secp521r1` but I only included `RSA_2048 | EC_prime256v1 | EC_secp384r1` since we can't import a Certificate in CloudFormation

Reply:
Personally I feel like that's the right thing to do for now too. Having the ability to create additional values probably makes up for any edge cases that could come up.